src/test/librbd/test_librbd.cc

```diff
@@ -4000,7 +4000,7 @@ TEST_F(TestLibRBD, TestCompareAndWriteMismatch)
   ASSERT_EQ(cmp_buffer.length(), written);
 
   // Compare should fail because of mismatch
-  uint64_t mismatch_off = 0;
+  uint64_t mismatch_off = -2;
```

Comment:
What is this change for?

Reply:
It was convention in the librados cmpext tests, but isn't required here. So got rid of.